web/src/main/java/com/epam/rft/atsy/web/controllers/rest/CandidateController.java

```diff
@@ -27,41 +29,45 @@
   public static final String NAME = "name";
   public static final String EMAIL = "email";
   public static final String PHONE = "phone";
+
+  private static final String EMPTY_STRING = StringUtils.EMPTY;
   private static final Logger LOGGER = LoggerFactory.getLogger(CandidateController.class);
   @Resource
   private CandidateService candidateService;
-  @Resource
-  private ObjectMapper objectMapper;
+
 
   @RequestMapping(method = RequestMethod.GET)
   public Collection<CandidateDTO> loadPage(
       @RequestParam(value = "filter", required = false) String filter,
       @RequestParam("order") String order,
       @RequestParam("sort") String sortField) {
 
+    SortingRequest.Order orderDirection = SortingRequest.Order.valueOf(StringUtils.upperCase(order));
+    SortingRequest.Field fieldName = SortingRequest.Field.valueOf(StringUtils.upperCase(sortField));
+
     FilterRequest filterRequest = FilterRequest.builder()
-        .order(SortingRequest.Order.valueOf(StringUtils.upperCase(order)))
-        .fieldName(sortField)
+        .order(orderDirection)
+        .fieldName(fieldName)
         .searchOptions(parseFilters(filter))
         .build();
 
     return candidateService.getAllCandidate(filterRequest);
   }
 
   private SearchOptions parseFilters(String filterJson) {
-    Map<String, String> filterMap = new HashMap<>();
-    if (StringUtils.isNotBlank(filterJson)) {
+    if (filterJson != null && StringUtils.isNotEmpty(filterJson) && !filterJson.equals("{}")) {
```

Comment:
StringUtils.isNotEmpty already contains a nullcheck, so you don't need to add one.